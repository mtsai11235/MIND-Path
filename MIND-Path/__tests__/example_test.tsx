import { render } from '@testing-library/react-native';

import Example, { CustomText } from '@/app/example';

describe('<Example />', () => {
  test('Text renders correctly on Example Page', () => {
    const { getByText } = render(<Example />);

    getByText('Welcome!');
  });

  test('CustomText renders correctly', () => {
    const tree = render(<CustomText>Some text</CustomText>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
