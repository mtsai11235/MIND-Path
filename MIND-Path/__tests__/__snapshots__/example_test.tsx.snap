// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> CustomText renders correctly 1`] = `
<Text>
  Some text
</Text>
`;
